applications/FluidDynamicsHydraulicsApplication/python_scripts/navier_stokes_two_fluid_hydraulic_solver.py

```diff
@@ -658,9 +680,9 @@ def _CreateScheme(self):
             domain_size + 1)
 
         return scheme
-    
+
     def _HydraulicBoundaryConditionCheck(self,boundary,name):
-        # Check if the inlet and outl 
+        # Check if the inlet and outl
```

Comment:
```suggestion
        # Check if there are inlet and outlet
```
Lets correct it as it popped up.